sql/hive/src/test/scala/org/apache/spark/sql/hive/SQLBuilderTest.scala

```diff
@@ -50,10 +50,8 @@ abstract class SQLBuilderTest extends QueryTest with TestHiveSingleton {
          """.stripMargin)
     }
 
-    val actualSQL = maybeSQL.get
-
     try {
-      assert(actualSQL === expectedSQL)
+      assert(generatedSql === expectedSQL)
```

Comment:
`generatedSQL`? Do we have any naming rules related to acronyms (`Sql` or `SQL`)?